### ISSUE TITLE
Fast Forward command disabled

### ISSUE THREAD

**Issue by khowling** [oeuillot/upnpserver#64]
great repo! I have a feature request: When using a smart TV, the ff and rewind buttons are not active on this server, same video on my other media server, the buttons work, you can press ff once to get x2, twice to get x5 etc.

**Comment by oeuillot** [comment]
Hello,
Thanks for your comment !
I don't know how to enable these buttons, I think it is a DNLA protocol
problem, but there are no public specifications :-(
If you have a link or a document which can help me, I will be very pleased.

Regards,
Olivier

2016-07-28 11:33 GMT+02:00 khowling <notifications@github.com>:

**Comment by khowling** [comment]
I'll look around, it looks like a 'seek' action, but, i cant see any detail on it.  By the way, it works perfectly on the other 6 devices in my house, as they all present a navigation timeline bar (whatever that is called), its just this samsung smart TV that only has the FF/RW buttons in its player.

**Comment by s-leger** [comment]
Hi,
maybe upnp miss an action or action params are not the right one (eg: seek byte based or seek time based).

Did you have anotner server where your smart tv work ?
Can you get the xml's for the working server ?

[You may download dlna spec from here ](http://www.dlna.org/guidelines/)

**Comment by khowling** [comment]
yes, the other server is twonky, its pre-installed on a zyxel nas, I've opened telnet to it & set max logging in twonkyserver.ini settings file - but i still cannot see the xml

**Comment by s-leger** [comment]
You may get it from:
your NAS URL/TMSContentDirectory.xml
